### ISSUE TITLE
Illegible Text Color With Dark Theme

### ISSUE THREAD

**Issue by smenes** [nextcloud/server#22377]
> 

### Steps to reproduce
1. Activate dark theme

### Actual behaviour
Illegible text color like this:
![jvJV1yB](https://user-images.githubusercontent.com/4074396/91002964-f61ddd80-e59d-11ea-89ae-ced4f9bd63be.png)
https://i.imgur.com/jvJV1yB.png

### Server configuration

**Operating system:**
Debian 10

**Web server:**
Nginx

**Database:**
MaraDB

**PHP version:**
7.4

**Nextcloud version:** (see Nextcloud admin page)
19.0.1

**Updated from an older Nextcloud/ownCloud or :fresh install**
fresh install

**Where did you install Nextcloud from:**
https://nextcloud.com/install/#instructions-server

**Comment by axheli** [comment]
Yes i can confirm this bug.

**Comment by szaimen** [comment]
I am not sure, but could it be possible that you are using a chromium based browser?
Then you could try to clear the cache and trigger a hard-reload after switching to dark mode by pressing the ctrl button + clicking on the reload button of your browser.

**Comment by smenes** [comment]
@szaimen I have already clear the cache before posting this report. and it does the same with Firefox anyway.
this bug come back time to time.

https://github.com/nextcloud/server/issues/11441
https://github.com/nextcloud/server/issues/14003
https://github.com/nextcloud/server/issues/18528
https://github.com/nextcloud/server/issues/10457

**Comment by szaimen** [comment]
hm strange. For me it works after clearing the browser cache.

**Comment by skjnldsv** [comment]
Please do the following steps:
1. [clear your browser's cache](https://www.refreshyourcache.com/)
2. [run](https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/occ_command.html#run-occ-as-your-http-user) an [`occ maintenance:repair`](https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/occ_command.html#maintenance-commands-label) on your server

Thanks :wink:

**Comment by skjnldsv** [comment]
duplicate of https://github.com/nextcloud/server/issues/21069

**Comment by tobi614** [comment]
> 
> 
> Please do the following steps:
> 
>     1. [clear your browser's cache](https://www.refreshyourcache.com/)
> 
>     2. [run](https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/occ_command.html#run-occ-as-your-http-user) an [`occ maintenance:repair`](https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/occ_command.html#maintenance-commands-label) on your server
> 
> 
> Thanks 😉

I have done that and cleared my Firefox Cache. But nothing was to repair. The font is after log out and log in still black.

![Occ](https://user-images.githubusercontent.com/64694684/92601621-5ce50b80-f2ad-11ea-83e4-46d233a75b4e.png)

**Comment by skjnldsv** [comment]
Let's continue on https://github.com/nextcloud/server/issues/21069 :)
